articles/mongodb-nodejs-custom-ranking/index.md

```diff
@@ -0,0 +1,204 @@
+### How to perform custom ranking for records from a MongoDB database in Node.js
+
+![Hero image](/engineering-education/mongodb-nodejs-custom-ranking/hero.jpg)
+
+#### Introduction
```

Comment:
You could start the markdown header from `###`. (The heading is ignored - it will be written in the front matter)

Reduce the markdown header by removing one `#` for all the subsequent headings